packages/datetime/src/datePickerCaption.tsx

```diff
@@ -20,7 +20,8 @@ export interface IDatePickerCaptionProps {
     onYearChange?: (year: number) => void;
 
     // normally we could extend ReactDayPicker.CaptionElementProps,
-    // but we don't want to introduce a typing dependency, so manually add props here
+    // but we don't want to introduce a typing dependency, so manually add props here.
```

Comment:
aha so this is no longer a problem! we can use types from `react-day-picker` because our consumers will be able to resolve them without installing an additional `@types` package. let's try to extend `CaptionElementProps` here (there's no rush for this PR as-is).